sphinx/builders/latex.py

```diff
@@ -211,6 +211,11 @@ def finish(self):
             logger.info(bold('copying images...'), nonl=1)
             for src, dest in iteritems(self.images):
                 logger.info(' ' + src, nonl=1)
+                subdir, _filename = path.split(dest)
```

Comment:
Sorry, you should not make sub directories here. The following process of LaTeX builder expects flatten layout.
If you put images under the sub directories, building PDF might fail for Japanese docs (see `sphinx/texinputs/Makefile_t`)
```
%.png %.gif %.jpg %.jpeg: FORCE_MAKE
    extractbb '$@'
```